pyqtgraph/widgets/TableWidget.py

```diff
@@ -311,22 +310,22 @@ def serialize(self, useSelection=False):
         if self.horizontalHeadersSet:
             row = []
             if self.verticalHeadersSet:
-                row.append(asUnicode(''))
+                row.append('')
             
             for c in columns:
-                row.append(asUnicode(self.horizontalHeaderItem(c).text()))
+                row.append(str(self.horizontalHeaderItem(c).text()))
```

Comment:
`QTableWidgetIem::text` returns a string, so we can ditch the casting here as well